common_rest/src/main/java/uk/gov/dhsc/htbhf/requestcontext/RequestContextConfiguration.java

```diff
@@ -77,7 +80,9 @@ public ObjectMapper objectMapper() {
         return new ObjectMapper()
                 .findAndRegisterModules()
                 .registerModule(javaTimeModule)
-                .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+                .configure(MapperFeature.DEFAULT_VIEW_INCLUSION, false)
```

Comment:
Did you know that `configure` taking in a `MapperFeature` is deprecated? Can we look for an alternative?

Reply:
Updated to the new format

Reply:
Ok, but shouldn't we now remove `.configure(MapperFeature.DEFAULT_VIEW_INCLUSION, false)`?

Reply:
You might notice that we're now configuring that on the JsonMapper, not the ObjectMapper